core/gesture.ts

```diff
@@ -172,13 +172,18 @@ export class Gesture {
   protected isEnding_ = false;
   private healStack_: boolean;
 
+  /** The event that most recently updated this gesture. */
+  private mostRecentEvent_: Event;
+
   /**
    * @param e The event that kicked off this gesture.
    * @param creatorWorkspace The workspace that created this gesture and has a
    *     reference to it.
    */
   constructor(
       private e: Event, private readonly creatorWorkspace: WorkspaceSvg) {
+    this.mostRecentEvent_ = e;
```

Comment:
Why was this not set before?

Reply:
The automatic migration created a parameter property, and renamed everything to `this.e`.

With your comment, I see I forgot to remove the `private` from `private e: Event`. Will check for similar misses & fix.